src/libs/navigateAfterJoinRequest/index.ts

```diff
@@ -3,7 +3,7 @@ import ROUTES from '@src/ROUTES';
 
 const navigateAfterJoinRequest = () => {
     Navigation.goBack(undefined, false, true);
-    Navigation.navigate(ROUTES.ALL_SETTINGS);
+    Navigation.navigate(ROUTES.SETTINGS);
```

Comment:
On mWeb, pressing back will go back to the report, but on native, pressing back will show the settings page. If we want to follow the mWeb, then we can remove this as I suggested in my proposal too.

Reply:
@bernhardoj If we change this for mWeb to be consistent with native, will this also affect desktop web? It would be nice to have mWeb and native beahviours to be the same, but only if it doesn't affect the desktop web behaviour.

Reply:
@Julesssss Any thoughts on this behaviour?

Reply:
Yes, except if we conditionally call the navigation in index.web.js.
```
if (Browser.isMobile()) {
    navigate to settings;
}
```

Reply:
Ok, this could work. Let's wait for @Julesssss' input on this.

Reply:
It's not going to be idea, but can we achieve this with the use of a mobile/mWeb specific file instead of the conditional?

Reply:
> mobile/mWeb specific file

I never see we have a mWeb specific file, so I'm not sure if that's possible. Actually, we can use getIsSmallScreenWidth instead of Browser.isMobile

Reply:
Oh right. Yeah that seems a good workaround to the rule 👍

Reply:
Updated for both index.web and index.desktop.